docs/controllers/services/annotations.md

```diff
@@ -17,7 +17,7 @@ If no custom name is specified, a default name is chosen consisting of the chara
 
 ## service.beta.kubernetes.io/do-loadbalancer-protocol
 
-The default protocol for DigitalOcean Load Balancers. Options are `tcp`, `http`, `https`, and `http2`. Defaults to `tcp`.
+The default protocol for DigitalOcean Load Balancers. Options are `tcp`, `udp`, `http`, `https`, and `http2`. Defaults to `tcp`. Note that if UDP is supported a healtcheck configuration configured against a TCP/HTTP/HTTPS service is still required. In order to use UDP protocol with a Load Balancer, please reach out to support with a ticket to enable it.
```

Comment:
> With a TCP health check port now required when UDP is used
It doesn't have to be TCP healthcheck. We support HTTP/HTTPS for the healthcheck with a path as well. We just don't support UDP as a healthcheck protocol.

Reply:
Sorry, I meant TCP as in the Kubernetes Service protocol, not as in the protocols that DO LBs support.

We currently pick the first Service (node) port for health checking, so my suggestion is to pick the first where the Service protocol is TCP. Otherwise, if no health check port is defined explicitly, I think our logic could theoretically pick a port from a UDP Service which I suppose would lead to an API error.

Reply:
Ah I see. I'd add some checks

Reply:
Fixed this!